Makefile

```diff
@@ -0,0 +1,9 @@
+.PHONY: build
+
+CARGO ?= cargo
+
+build: ## Build the project
+	$(CARGO) build
+
+build-watch: ## Build the project (continuously)
+	$(CARGO) watch -- $(MAKE) build
```

Comment:
Why not just run `cargo watch` with defaults? In any case, that's local developer tooling, which does not belong in this repo